Nbuild/Program.cs

```diff
@@ -99,6 +107,84 @@ static int Main(string[] args)
         return (int)result.Code;
     }
 
+    /// <summary>
+    /// Run the build targets based on the provided arguments.  This method is added for convenience so that 
+    /// nb.exe can be run from the command line without having to specify the command line options.
+    /// </summary>
+    /// <param name="args">The command line arguments.</param>
+    /// <param name="options">The parsed command line options.</param>
+    /// <returns>A ResultHelper object indicating the result of the build process. 
+    ///     Otherwise, int.MaxValue -1 to indicate that target is not running</returns>
+    private static ResultHelper RunBuildTargets(string[] args, out Cli options)
+    {
+        options = new Cli();
+        var verbose = options.Verbose;
+        string? target = null;
+
+        switch (args.Length)
+        {
+            case 0:
+                options = new Cli() { Verbose = true };
+
+                return BuildStarter.Build(target, verbose);
+         
+            case 1 when args[0].Equals(CmdHelp, StringComparison.InvariantCultureIgnoreCase):
+                return ResultHelper.Success("");
+            
+            case 1 when !args[0].Contains(CmdHelp, StringComparison.InvariantCultureIgnoreCase) && !args[0].StartsWith("-"):
+                target = args[0];
+                return BuildStarter.Build(target, verbose);
+
+            case 3 when args[1].Equals("-v", StringComparison.InvariantCultureIgnoreCase) && (bool.TryParse(args[2], out verbose)):
+                options.Verbose = verbose;
+                target = args[0];
+                return BuildStarter.Build(target, verbose);
+            
+            default:
+                return ResultHelper.Fail(int.MaxValue, "Display Help");
+        }
+
+        //if (args.Length == 1)
```

Comment:
What is the status of this commented-out code?

Reply:
Good catch!